packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -60,7 +60,10 @@ const CreateAdForm = () => {
         my_ads_store.setShowAdForm(false);
     };
 
+    React.useEffect(() => {}, [my_ads_store.current_method.is_deleted]);
```

Comment:
needed empty useEffect to force the component to re render when the method changes. tried to use reactions but it didn't rerender the component. This is for validating the form when the user is editing a payment method